tensorflow/core/ops/array_ops.cc

```diff
@@ -2292,6 +2292,24 @@ Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
   return Status::OK();
 }
 
+Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
+                   ShapeHandle sizes_value,
+                   std::vector<DimensionHandle>* dims) {
+  for (int i = 0; i < c->Rank(sizes_value); ++i) {
+    DimensionHandle dim = c->Dim(sizes_value, i);
+    DimensionHandle begin = c->Dim(begin_value, i);
+    if (c->ValueKnown(begin) && c->ValueKnown(dim)) {
+      // If both `begin` and `size` are known, processing:
+      DimensionHandle result;
+      TF_RETURN_IF_ERROR(c->Subtract(dim, c->Dim(begin_value, i), &result));
```

Comment:
c->Dim(begin_value, i)   can be replaced with   begin

Reply:
Thanks @cwhipkey. The PR has been updated.